keyboards/nopunin10did/kastenwagen48/keymaps/jonavin/kastenwagen.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
This appears to be a VIA json. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

> Do not include VIA json files in the PR. These do not belong in the QMK repository as they are not used by QMK firmware – they belong in the [VIA Keyboard Repo](https://github.com/the-via/keyboards>

Reply:
It was in the .ignore file but somehow got into this  pr. It’s been removed from the folder.